po/extra/pt.po

```diff
@@ -0,0 +1,359 @@
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:56
+msgid "Special thanks to eudaldgr for porting Eddy to the Meson build system."
+msgstr ""
+"Um agradecimento especial ao eudaldgr por ter portado o Eddy para o sistema "
+"de compilação Meson."
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:62
+msgid ""
+"This release fixes small annoyances with the UI and includes new "
+"translations:"
+msgstr ""
+"Esta versão corrige pequenos constrangimentos com a IU e inclui novas "
+"traduções:"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:64
+msgid "Brazilian Portuguese translation (Paulo Galardi)"
+msgstr "Tradução do Português do Brasil (Paulo Galardi)"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:65
+msgid "Lithuanian translation (welaq)"
+msgstr "Tradução lituana (welaq)"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:66
+msgid "Spanish translation (Juan Carlos)"
+msgstr "Tradução espanhola (Juan Carlos)"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:67
+msgid "French translation (nvivant)"
+msgstr "Tradução francesa (nvivant)"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:71
+msgid ""
+"Fast switching to the detailed view while loading packages caused "
+"unintentional auto-switching to list view"
+msgstr ""
+"Mudança rápida para a vista detalhada durante o carregamento de pacotes "
+"causou a comutação automática não intencional para a vista de lista"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:72
+msgid "Error dialog now has a proper title"
+msgstr "A caixa de diálogo de erro agora tem um título adequado"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:78
+msgid ""
+"This release adds small improvements to the user interface and adds initial "
+"support for translations."
+msgstr ""
+"Esta versão adiciona pequenas melhorias à interface do utilizador e "
+"adiciona suporte inicial para traduções."
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:79
+#: data/com.github.donadigo.eddy.appdata.xml.in:96
+msgid "Miscellaneous:"
+msgstr "Diversos:"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:81
+msgid "Show operation percentage alongside the status label"
+msgstr "Mostrar a percentagem de operação ao lado da descrição do estado"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:82
+msgid "Properly handle terminating the application by external processes"
+msgstr "Tratar adequadamente o terminar da aplicação por processos externos"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:83
+msgid "Fix punctuation in the quit dialog"
+msgstr "Corrige a pontuação na caixa de diálogo de saída"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:84
+msgid "Replace confusing quit dialog actions with more explicit ones"
+msgstr ""
+"Substitui as ações confusas da caixa de diálogo de saída por outras mais "
+"explícitas"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:85
+#: data/com.github.donadigo.eddy.appdata.xml.in:98
+msgid "Update application metadata"
+msgstr "Atualizar os metadados da aplicação"
```

Comment:
For consistency, this should be either "Atualiza os..." or "Atualização dos..."

Reply:
Fixed